homeassistant/components/fyta/__init__.py

```diff
@@ -0,0 +1,82 @@
+"""Initialization of FYTA integration."""
+from __future__ import annotations
+
+from datetime import datetime
+import logging
+
+from fyta_cli.fyta_connector import FytaConnector
+from fyta_cli.fyta_exceptions import (
+    FytaAuthentificationError,
+    FytaConnectionError,
+    FytaPasswordError,
+)
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+from homeassistant.helpers.device_registry import DeviceEntry
+
+from .const import DOMAIN
+from .coordinator import FytaCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = (Platform.SENSOR,)
```

Comment:
Please make it a list

Reply:
do you mean "Platform.SENSOR,"  on a new line?

Reply:
No, it's a tuple now because of the `()` but you should use `[]`